jp/components/grid_search.md

```diff
@@ -170,14 +170,14 @@ public updateSearch() {
 }
 ```
 
-#### Persistence
+#### 永続化
 
-What if we would like to filter and sort our grid or even to add and remove records? After such operations, the highlights of our current search automatically update and persist over any text that matches the **searchText**! Furthermore, the search will work with paging and will persist the highlights through changes of the grid's `perPage` property.
+グリッドのフィルターや並べ替え、レコードの追加や削除などの場合、処理後に現在の検索が自動的に更新されて **searchText** に一致するテキストが保持されます。更に検索がページングで動作し、グリッドの `perPage` プロパティの変更時も強調表示が保持されます。
 
-#### Adding icons
+#### アイコンの追加
 
-By using some of our other components, we can create an enriched user interface and improve the overall design of our entire search bar! We can have a nice search or delete icon on the left of the search input and some material design icons combined with nice ripple styled buttons for our search options and navigation on the right. We can wrap these components inside an input group for a more refined design.
-To do this, let's go and grab the [**IgxInputGroup**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/input_group.html), [**IgxIcon**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/icon.html),  [**IgxRipple**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/ripple.html) and the [**IgxButton**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/button.html) modules.
+その他のコンポーネントを使用するためにユーザー インターフェイスを作成し、検索バー全体のデザインを向上します。左側にはより洗練されたデザイン検索または削除アイコン、右側には検索オプション用に Ripple スタイルのマテリアル デザイン アイコンを表示できます。入力グループ内のコンポーネントをラップしてより洗練されたデザインにすることができます。
```

Comment:
I'm confused here.  I think this is missing "navigation"?  Navigation is also on the right side.  The ripple style is applied to the search options buttons.

Reply:
Updated whole sentence.